googletest/docs/advanced.md

```diff
@@ -686,7 +686,7 @@ Note that a death test only cares about three things:
 2.  (in the case of `ASSERT_EXIT` and `EXPECT_EXIT`) does the exit status
     satisfy `predicate`? Or (in the case of `ASSERT_DEATH` and `EXPECT_DEATH`)
     is the exit status non-zero? And
-3.  does the stderr output match `regex`?
+3.  does the stderr output match `matcher`?
```

Comment:
This has to be a regex, right? Not any matcher?

Reply:
Ok, pardon me ;)